ably/ably-ui
# [MAW-510] Add new G2 badges to our footer

Jira Ticket Link / Motivation
----
https://ably.atlassian.net/browse/MAW-510



Summary of changes
----
Update badges in the footer component 



How do you manually test this?
----
Check badges in the footer component
![image](https://github.com/ably/ably-ui/assets/987004/a8e1bb47-2b6c-437b-83b8-7f2a573778aa)



Reviewer Tasks (optional)
----




Merge/Deploy Checklist
----




Frontend Checklist
----
